src/user_templates_api/templates/jupyter_lab/templates/squidpy/metadata.json

```diff
@@ -8,5 +8,13 @@
     ], 
     "is_multi_dataset_template": true,
     "template_format": "jinja",
+    "examples": [
+        {
+            "example_id": 0,
```

Comment:
What is your envisioned purpose for `example_id`?

Reply:
I figure that if there are multiple examples, we might want to prioritise some in ordering, that we would want to point people to first

Reply:
I'm fine with a field to order them or we can just rely on the order of the array, your call!